add-ons/che/che.addon

```diff
@@ -0,0 +1,26 @@
+# Name: che
+# Description: Setup and Configure Eclipse Che Template and Image Streams
+# Url: https://www.eclipse.org/che/docs/setup/openshift/index.html
+
+echo [CHE] Create Image Stream
+oc apply -f streams/che-server-streams.json -n openshift
+
+echo [CHE] Create the Che server Template
+oc apply -f templates/che-single-user.yml -n openshift
+
+echo [CHE] Creating mini-che project
+oc new-project mini-che --description="Eclipse Che on minishift" --as=developer
+
+echo [CHE] Switching to mini-che...
+oc project mini-che
+
+echo [CHE] Deploying Che on minishift (image docker.io/eclipse/che-server:nightly)
```

Comment:
Just a reminder to remove `nightly` tag. Probably use exact version.

Reply:
@sunix - i have one query around project creation here. say if the user deletes the project then he might not have this instance so he has to create via `Add To Project` or reapply the addon( am not sure this is possible). WDYT?